.github/workflows/build-images-for-tag-release.yaml

```diff
@@ -21,7 +21,6 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v4
       - name: Set environment variables
-        id: set_env_variables
```

Comment:
General question: Why have some steps with an id and others like this one without an id?

Reply:
`id` is used when you want to reference it from another step, for instance if generates some outputs. It is harmless anyway.